lib/pengine/pe_actions.c

```diff
@@ -1203,7 +1203,7 @@ node_priority_fencing_delay(const pcmk_node_t *node,
     GList *gIter = NULL;
 
     // PCMK_OPT_PRIORITY_FENCING_DELAY is disabled
-    if (scheduler->priority_fencing_delay <= 0) {
+    if (scheduler->priv->priority_fencing_ms <= 0U) {
```

Comment:
could never be `< 0` but no need to change now

Reply:
actually static analysis sometimes complains about that, updated commit